docs/modules/reference/pages/provisioning/detectors/ActiveMQDetector.adoc

```diff
@@ -11,37 +11,37 @@ Use this detector to find and assign services based on ActiveMQ.
 
 == Configuration and use
 
```

Comment:
Suggest adding the following to every detector that uses any of the common configuration parameters. (I like what you did in the common config. params table with notes for the detectors that are exceptions to those parameters.)

This monitor implements the Common Configuration Parameters.

(With a link back to the table in the detectors intro. doc.)